python/hail/methods/statgen.py

```diff
@@ -4,7 +4,7 @@
 
 import hail as hl
 import hail.expr.aggregators as agg
-from hail.expr.expressions import *
+from hail.expr.update_entries_expressions import *
```

Comment:
something has gone wrong

Reply:
_sigh_ `find and replace in selection` has failed me. will fix